PCGen-Formula/code/src/java/pcgen/base/formula/base/StaticStrategy.java

```diff
@@ -45,6 +45,6 @@ public void addVariable(DependencyManager mgr, String varName)
 	public static VariableID<?> getVariableID(DependencyManager mgr, String varName)
 	{
 		VariableLibrary varLib = mgr.get(DependencyManager.FMANAGER).getFactory();
-		return varLib.getVariableID(mgr.get(DependencyManager.INSTANCE), varName);
+		return varLib.getVariableID(mgr.get(DependencyManager.INSTANCE).get(), varName);
```

Comment:
This seems off: there is no .isPresent check. Should we pass this through?

Reply:
Made that item not Optional anymore